users/templates/user_register.html

```diff
@@ -0,0 +1,13 @@
+{% extends "index.html" %}
+{% block title %}"РтК" Реєстрація{% endblock %}
+{% block container %}
+    <br>
+    <h2>Введіть логін та пароль</h2>
+    <br>
+    <form method="post">
+        {% csrf_token %}
+        <input id="username" name="username" type="text">
+        <input id="password1" name="password1" type="password">
+        <input id="password"2 name="password2" type="password">
+        <input type="Submit" value="Реєструватися">
+{% endblock %}
```

Comment:
Треба закрити тег форми

```suggestion
</form>
{% endblock %}
```